src/core_plugins/kibana/public/dashboard/dashboard_config.js

```diff
@@ -0,0 +1,22 @@
+import { uiModules } from 'ui/modules';
+uiModules.get('kibana')
+  .provider('dashboardConfig', () => {
+    let hideWriteControls = false;
+
+    return {
+      /**
+       * Part of the exposed plugin API - do not remove without careful consideration.
+       * @type {boolean}
+       */
+      turnHideWriteControlsOn() {
+        hideWriteControls = true;
+      },
+      $get() {
+        return {
+          getHideWriteControls() {
```

Comment:
Personally, my opinion is that `can` and `should` are pretty ambiguous because whether something "can" be or "should" be is pretty relative and open to interpretation. `getIs` also feels a little strange to me when I read it. I generally just stick with `is` and `are` where possible, e.g. `areWriteControlsVisible` or `writeControls.isVisible`. But I don't think we have an agreed-upon convention, so until we have that I think the most efficient way forward is if we each just write names in ways that we're comfortable with and, to Stacey's point, not get too hung up on nitpicking the way other people write them.